telemetry/convert.py

```diff
@@ -221,7 +221,7 @@ def convert_saved_session(self, json_dict):
 
         # WINNT is reported as Windows_NT in the unified ping, and apparently
         # we have some Win95 users
-        if info.get("OS") == "Windows_NT" or info.get("OS") == "Windows_95":
+        if info.get("OS").startswith("Windows_"):
```

Comment:
Sorry, we should also guard for `None`, so let's use `info.get("OS", "Other").startswith("Windows_"):`